### DISCUSSION TITLE
Manual Refresh Button

### DISCUSSION THREAD

**Discussion by connorwinston** [directus/directus#15973] [Feature Requests]
After using Directus for a very long time, one of my most wished-for items was a Manual refresh button in the following places.

- Collection Layout
- Insight Dashboard
- Flows Logs

I find my self almost every day either refreshing the whole page (which takes substantially longer) or clicking out and back into a page to get current data.

Does anyone have any thoughts on this?

**Comment by benhaynes** [comment]
While I agree that a soft refresh button would be nice now, I think real time websockets would be ideal... since data would just be "always current". Though maybe there's a car for keeping it static until you want to re-fetch?

**Comment by rijkvanzanten** [comment]
Also semi related to https://github.com/directus/directus/discussions/15854

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
